gitops/triggers/triggers-crds/interceptors/kustomization.yaml

```diff
@@ -1,5 +1,8 @@
 resources:
-  #interceptor files
+  # custom resources definitions
+  - deployments.apps.yaml
+  - services.yaml
```

Comment:
Same as the pod.yaml. Do we need these files?
I think with the new syncer mode, we just need to specify deployments and services under resources to sync in the syncer.yaml file. Please verify.

Reply:
Right now, removing those resources from kustomization.yaml ends up prevent the deployment of `triggers-interceptors`. I'm not quite sure, but I believe that this is a side effect of the controllers running on KCP at the moment instead of plnsvc (the controllers need the CRDs to create the resources in KCP, not the KCP version of CRDs). Moving the controllers out of KCP is something we'll tackle as a separate story.
I did move the `pods.yaml` to reflect the fact that it's needed by this application.